host/tests/integration_tests/python/test_utils.rs

```diff
@@ -0,0 +1,34 @@
+use datafusion_common::DataFusionError;
+use datafusion_udf_wasm_host::{WasmComponentPrecompiled, WasmScalarUdf};
+use tokio::sync::OnceCell;
+
+static COMPONENT: OnceCell<WasmComponentPrecompiled> = OnceCell::const_new();
+
+async fn python_component() -> &'static WasmComponentPrecompiled {
+    COMPONENT
+        .get_or_init(async || {
+            let wasm_binary = tokio::fs::read(format!(
+                "{}/../target/wasm32-wasip2/debug/datafusion_udf_wasm_python.wasm",
```

Comment:
Does this just work out-of-the-box? Or do you need to like `cargo build --target wasm32-wasip2` first?